docs/application/web/getting-started/wearable/first-app.md

```diff
@@ -235,15 +235,15 @@ To run the application on the emulator:
 
         ![Application running in the emulator](media/emulator_running_ww.png)
 
-        > **Note** 
+        > **Note**  
 		> If the emulator display has switched off, you cannot see the application launch. To switch the display on, click the **Power** key (in the lower-right corner of the emulator).
 
         While the application is running, the **Log** view in the Tizen Studio shows the log, debug, and exception messages from the methods defined in the log macros. To see the view, in the Tizen Studio menu, go to **Window &gt; Show View &gt; Log**.
 
 For more information on using the emulator features, see [Using Emulator Control Keys, Menu, and Panel](../../../tizen-studio/common-tools/emulator-control-panel.md) and [Using Extended Emulator Features](../../../tizen-studio/common-tools/emulator-features.md).
 
 <a name="target"></a>
-### Running on a Target Device 
+### Running on a Target Device
 
 To run the application on a target device:
 
```

Comment:
Don't we need <a name="remote_device"></a> in this page on line 270?

Reply:
already done